heartbeat/monitors/active/dialchain/builder.go

```diff
@@ -199,15 +197,5 @@ func makeEndpointJobs(
 	if err != nil {
 		return nil, err
 	}
-	return []monitors.Job{monitors.WithJobId(jobID, monitors.WithFields(fields, job))}, nil
-}
-
-func jobID(typ, jobType, host string, ports []uint16) string {
```

Comment:
I see there was some value in the old id. I wonder if this is still something we can reconstruct on the UI side as all the data should be available there if needed?

Reply:
Yes, exactly. I don't think we're missing it inside of heartbeat itself.